src/components/Text/variants.ts

```diff
@@ -116,7 +116,6 @@ export const variants = {
       fontStyle: 'normal',
       fontWeight: 600,
       lineHeight: '2rem',
-      whiteSpace: 'nowrap',
```

Comment:
Well it separates some long labels onto multiple lines on small screens, but I think that it's better that way. 🤔 I didn't notice anything serious with our input fields.

Reply:
Yeah, if you checked its probably ok. But i have a feeling its there for a reason. YOu added this line with commit https://github.com/sos-lapsikyla/ylitse-web/commit/b716abe8562dd13d85ce6892043fcc61bd3e9f6e
"Add (absence) slider component"

Reply:
It didn't have anything to do with the slider, but the labels with long names. For example, the password input labels.
 
This is how some fields with long labels look now: <img width="249" alt="Näyttökuva 2024-10-24 kello 11 39 18" src="https://github.com/user-attachments/assets/92ca9402-d238-4bfe-af20-1ebc2a6c741a">

And here is the current master branch with 'nowrap': <img width="265" alt="Näyttökuva 2024-10-24 kello 11 39 50" src="https://github.com/user-attachments/assets/fa126af4-4c0c-440d-8ff8-31d8d9c8e3f3">

This only happens with the tiniest screen sizes, but I do think that wrapping onto multiple lines is better, because the _Show password_ buttons do that too. 🤔

Reply:
Yeah, its better imo too